torch/_dynamo/variables/builtin.py

```diff
@@ -712,6 +712,20 @@ def _make_handler(fn, arg_types: List[type], has_kwargs: bool):
                 tx, [v.realize() for v in args], kwargs
             )
 
+        if inspect.isclass(fn) and issubclass(fn, Exception):
+
+            def create_exception_class_object(tx, args, kwargs):
+                if fn is AssertionError and not all(
```

Comment:
This is just to maintain the status quo for test `pytest test/dynamo/test_repros.py::ReproTests::test_rewrite_assert_with_non_string_msg`